ShowerShow/Control/BlobStorageController.cs

```diff
@@ -0,0 +1,261 @@
+        }
+
+
+        [Function("UploadProfilePicture")] // USE POSTMAN TO TEST
+        [OpenApiParameter(name: "UserId", In = ParameterLocation.Path, Required = true, Type = typeof(Guid), Description = "The User ID parameter")]
+        [OpenApiOperation(operationId: "UploadProfilePicture", tags: new[] { "BlobStorage" })]
+        public async Task<HttpResponseData> UploadProfilePicture([HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "user/{UserId:Guid}/profile/uploadpic")] HttpRequestData req, Guid UserId)
+        {
+            _logger.LogInformation("Uploading to blob.");
+
+            string containerName = Environment.GetEnvironmentVariable("ContainerProfilePictures");
+            HttpResponseData responseData = req.CreateResponse();
+            try
+            {
+                var parsedFormBody = MultipartFormDataParser.ParseAsync(req.Body);
+                var file = parsedFormBody.Result.Files[0];
+
+
+                CloudBlobClient client = account.CreateCloudBlobClient();
+                CloudBlobContainer container = client.GetContainerReference(containerName);
+                await container.CreateIfNotExistsAsync();
+
+                CloudBlockBlob blockBlob = container.GetBlockBlobReference(UserId.ToString() + ".png");
+                blockBlob.Properties.ContentType = file.ContentType;
+
+                if (!blockBlob.Properties.ContentType.Contains("image"))
+                {
+                    responseData.StatusCode = HttpStatusCode.BadRequest;
+                    return responseData;
+                }
+
+                await blockBlob.UploadFromStreamAsync(file.Data);
+                responseData.StatusCode = HttpStatusCode.OK;
+                return responseData;
+            }
+            catch
+            {
+                responseData.StatusCode = HttpStatusCode.BadRequest;
+                return responseData;
+            }
+
+        }
+        [Function("GetProfilePictureOfUser")] // USE POSTMAN TO TEST
+        [OpenApiParameter(name: "UserId", In = ParameterLocation.Path, Required = true, Type = typeof(Guid), Description = "The User ID parameter")]
+        [OpenApiOperation(operationId: "GetProfilePictureOfUser", tags: new[] { "BlobStorage" })]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "image/jpeg", bodyType: typeof(byte[]), Description = "The OK response with the profile picture.")]
+        public async Task<HttpResponseData> GetProfilePictureOfUser([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "user/{UserId:Guid}/profile/getpic")] HttpRequestData req, Guid UserId)
+        {
+            _logger.LogInformation("Uploading to blob.");
+
+            string containerName = Environment.GetEnvironmentVariable("ContainerProfilePictures");
+            HttpResponseData responseData = req.CreateResponse();
+            try
+            {
+                CloudBlobClient client = account.CreateCloudBlobClient();
```

Comment:
Should you check whether the user exist or not? I'm not sure about the requirement of it